pkg/cosign/tuf/client.go

```diff
@@ -118,11 +141,11 @@ func (t *TUF) getRootStatus() (*RootStatus, error) {
 		return nil, errors.Wrap(err, "getting trusted meta")
 	}
 	for role, md := range trustedMeta {
-		expires, err := getExpiration(md)
+		mdStatus, err := getMetadataStatus(md)
 		if err != nil {
-			return nil, errors.Wrap(err, fmt.Sprintf("getting expiration for %s", role))
+			continue
```

Comment:
If there was any error, shouldn't this function return an error after iterating through all roles? Otherwise it'll return no error.

Reply:
I don't want to exit early and end up not printing out the full set of metadata statuses.

I could add an error string to `MetadataStatus` though!

Reply:
done, added an error in case we can't fetch status info